### ISSUE TITLE
🚀 Feature: Upload Speed and Percentage Stats

### ISSUE THREAD

**Issue by jasonelilewis81** [stonith404/pingvin-share#199]
### 🔖 Feature description

It would be great to see the upload speed, percentage of file upload completed, and possibly an estimate of the time left to upload each file. 

### 🎤 Pitch

When uploading small files that take a couple seconds I know seeing the stats don't really matter. But when uploading larger files it would really help to see how fast they are uploading and how much time is left.

**Comment by pierrbt** [comment]
This is an interesting idea, it would be easy to calculate the progression using a chunk counter

**Comment by codsec** [comment]
It would be nice!!!!

**Comment by stonith404** [comment]
@McFex mentioned in the issue #243 that the color of the progress indicator in dark mode is poor. This will be fixed too when I have the time to work on this.

**Comment by patrickasmus** [comment]
I think this is a great idea too!

**Comment by sahara101** [comment]
Yes, please! I keep trying to figure out from the small loading in dark mode if my 10GB upload is doing anything or not. Speed + % would be awesome.

**Comment by fcpwiz** [comment]
Yes! This is the last major feature missing. Any update on this feature request?

**Comment by umekzg** [comment]
+1
Large files are a nightmare to upload without upload stats....

**Comment by tueboesen** [comment]
I completely agree that this feature would be very nice to have. 
I am currently testing out pingvin share and one of the first things I tried was to upload a 10 GB file to it, to see how fast it seems to be. Currently it is still uploading as far as I can see, but my only indicator is a very small wheel that is getting slightly filled as time passes.

I have no way to see what my upload speed is, or expected time until it finishes.

So if I want to figure out whether it uploads at any reasonable speed I manually need to time when I started the upload and when it finishes and based on that I can then calculate the upload speed, which is of course not ideal.

The reason why I am curious about this is that I suspect that the transfer speeds are rather slow behind a reverse proxy unless I get everything set up just right.

Anyways the tool looks very nice otherwise :)
